src/RwaUrn.sol

```diff
@@ -45,18 +49,28 @@ library DSMathCustom {
     }
 
     /**
-     * @dev Converts `wad` (10^18) into a `rad` (10^45) by multiplying it by RAY (10^27).
+     * @dev Converts wei into a `wad` (10^18) by multiplying it by WAD (10^18).
      *  - wad: used for token balances
      *  - ray: used for interest rates
      *  - rad: used for Dai balances inside the Vat
      */
+    function wad(uint256 wei_) internal pure returns (uint256 z) {
+        return mul(wei_, WAD);
+    }
+
+    /**
+     * @dev Converts `wad` (10^18) into a `rad` (10^45) by multiplying it by RAY (10^27).
+     */
     function rad(uint256 wad) internal pure returns (uint256 z) {
         return mul(wad, RAY);
     }
 }
 
 /**
- * @title A vault for Real-World Assets (RWA).
+ * @author Kaue Cano <kaue@clio.finance>
+ * @author Lev Livnev <lev@liv.nev.org.uk>
+ * @title RwaUrn2: A capped vault for Real-World Assets (RWA).
+ * @dev This vault implements `gemCap`, the maximum amount of gem the urn can hold.
  */
 contract RwaUrn {
```

Comment:
We should probably rename this contract to `RwaUrn2`.
Don't forget to update the references to it as well.

Reply:
Why we should rename it?

Reply:
it's the pattern used in gem-joins https://github.com/makerdao/dss-gem-joins/blob/master/src/join-2.sol